tests/testthat/test-bind_tables.R

```diff
@@ -0,0 +1,49 @@
+# Load initial variables
+token <- Sys.getenv("REDCAPTIDIER_LONGITUDINAL_API")
+redcap_uri <- Sys.getenv("REDCAP_URI")
+
+test_that("bind_tables works with no specifications", {
+
+  read_redcap_tidy(redcap_uri, token) %>%
+    bind_tables()
+
+  expect_true(exists("nonrepeated", envir = global_env()))
```

Comment:
Is `envir = global_env()` necessary here?

Reply:
You're right, default behavior is to check global anyway. Removed 👍